staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -758,7 +758,7 @@ func (s *store) List(ctx context.Context, key string, opts storage.ListOptions,
 
 			data, _, err := s.transformer.TransformFromStorage(kv.Value, authenticatedDataString(kv.Key))
 			if err != nil {
-				return storage.NewInternalErrorf("unable to transform key %q: %v", kv.Key, err)
+				return storage.NewInternalErrorf("unable to transform key %q: %v, please check whether the secret has changed", kv.Key, err)
```

Comment:
The `transformer` interface is abstract, so although today it might be true that the cause of the failure was the changed secret, it's not guaranteed to stay this way. I think we should keep the error message general.

Reply:
@caesarxuchao I agress with you,  I don't know if it's suitable to specify the error message in subclass(cbc). I think the cbc should keep the error message geneal too. So, Can you give me some advice?

Reply:
What does the `err` contain? `err` is return by a concrete implementation of the transformer interface, I would expect `err` to contain more specific info.